Services/DataX.Flow/Flow.ManagementService/Controllers/FlowManagementController.cs

```diff
@@ -422,6 +422,25 @@ public async Task<ApiResult> RestartJob([FromBody] string jobName)
             }
         }
 
+        [HttpGet]
```

Comment:
Should we make it HttpPost. Will also help if take take an input list of jobnames to restart